scripts/openyurt_deployer/configs/system.go

```diff
@@ -0,0 +1,59 @@
+package configs
+
+import (
+	"runtime"
+)
+
+// System environment struct
+type SystemEnvironmentStruct struct {
+	GoInstalled                         bool
+	ContainerdInstalled                 bool
+	RuncInstalled                       bool
+	CniPluginsInstalled                 bool
+	SystemdStartUp                      bool
+	GoVersion                           string
+	GoDownloadUrlTemplate               string
+	ContainerdVersion                   string
+	ContainerdDownloadUrlTemplate       string
+	ContainerdSystemdProfileDownloadUrl string
+	RuncVersion                         string
+	RuncDownloadUrlTemplate             string
+	CniPluginsVersion                   string
+	CniPluginsDownloadUrlTemplate       string
+	KubectlVersion                      string
+	KubeadmVersion                      string
+	KubeletVersion                      string
+	Dependencies                        string
+	TmpDir                              string
+	CurrentOS                           string
+	CurrentArch                         string
+	CurrentDir                          string
+	UserHomeDir                         string
+	NodeHostName                        string
+}
+
+// Current system environment
+var System = SystemEnvironmentStruct{
+	GoInstalled:                         false,
+	ContainerdInstalled:                 false,
+	RuncInstalled:                       false,
+	CniPluginsInstalled:                 false,
+	SystemdStartUp:                      true,
+	GoVersion:                           "1.18.10",
```

Comment:
same for this config. also make sure the golang version is the same as in the CI